### PR TITLE
Fix resource and formula name case

### PR DESCRIPTION
**Pull request by robsonpeixoto**

### CODE REVIEW

File: poet/poet.py [hunk 1]
```diff
@@ -54,20 +62,18 @@ def install
...
     url "{{ resource.url }}"
     {{ resource.checksum_type }} "{{ resource.checksum }}"
   end
 """)
 
-
 class PackageNotInstalledWarning(UserWarning):
     pass
 
-
```
Comment (tdsmith) [inline poet/poet.py]: Two newlines is PEP 8 style; can you leave these in?

File: poet/poet.py [hunk 2]
```diff
@@ -18,14 +18,22 @@
...
-"""class {{ package.name|capitalize }} < Formula
+jinja_env = Environment(trim_blocks=True)
+
+def camel_case(s):
+    return ''.join(map(lambda s: s.capitalize(), re.split("-|_", s)))
+
+jinja_env.filters['camelcase'] = camel_case
+
+FORMULA_TEMPLATE = jinja_env.from_string(
+"""class {{ package.name|camelcase}} < Formula
```
Comment (tdsmith) [inline poet/poet.py]: :+1:

File: poet/poet.py [hunk 3]
```diff
@@ -40,7 +48,7 @@
   def install
 {% if resources %}
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python{{ py_version }}/site-packages"
-    %w[{{ resources|map(attribute='name')|join(' ') }}].each do |r|
+    %w[{{ resources|map(attribute='name')|map('lower')|join(' ') }}].each do |r|
```
Comment (tdsmith) [inline poet/poet.py]: Should we always downcase the resource names? I think we don't need to but maybe I'm forgetting something.

File: poet/poet.py (hunk 2 above)
Comment (tdsmith) [inline poet/poet.py]: Style nit: want to balance the whitespace inside the brackets on line 36?

**Comment by tdsmith** [comment]
Once this is ready I'll release 0.4.

File: poet/poet.py (hunk 3 above)
Comment (robsonpeixoto) [inline poet/poet.py]: I formulas how I have read, are using resources in lower case.

**Comment by robsonpeixoto** [comment]
I'll create a better PR

**Comment by tdsmith** [comment]
Thanks! (And thanks for keeping me honest on my indentation. :P)

**Pull request closed without merging**